util/uvmdvgen.py

```diff
@@ -104,12 +119,19 @@ def main():
         help=
         """Tests are now run with dvsim.py tool that requires a hjson based sim cfg.
              Setting this option will also result in the Makefile to be auto-generated (which is
-             the older way of building and running sims going through deprecation).""")
+             the older way of building and running sims going through deprecation)."""
+    )
 
     args = parser.parse_args()
     if args.agent_outdir == "name": args.agent_outdir = args.name
     if args.env_outdir == "name": args.env_outdir = args.name
 
+    if not args.has_ral:
+      if args.is_cip:
+        sys.exit("has_ral must be True for testbenches derived from CIP libraries")
+      if args.has_interrupts:
+        sys.exit("has_ral must be True for blocks that have interrupts")
+
```

Comment:
I think the formatting is off. Please run this through `util/lintpy.py`.

Reply:
I ran `lintpy.py`with the `--fix` option, think everything should be okay now.